Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -718,6 +734,29 @@ Function/WAVE SF_FormulaExecutor(jsonID, [jsonPath, graph])
 			endfor
 			Redimension/N=(-1, -1, j) out
 			break
+		case "labnotebook":
+			/// `labnotebook(string key, array sweeps)`
+			///
+			/// return lab notebook @p key for all @p sweeps
+			ASSERT(!ParamIsDefault(graph) && !IsEmpty(graph), "Graph not specified.")
+
+			JSONtype = JSON_GetType(jsonID, jsonPath + "/0")
+			ASSERT(JSONtype == JSON_STRING, "first parameter needs to be a string labnotebook key")
+			str = JSON_GetString(jsonID, jsonPath + "/0")
+
+			WAVE sweeps = SF_FormulaExecutor(jsonID, jsonPath = jsonPath + "/1", graph = graph)
+			ASSERT(DimSize(sweeps, COLS) < 2, "Sweeps are one-dimensional.")
+
+			WAVE numericalValues = GetLBNumericalValues(graph)
+			numIndices = DimSize(sweeps, ROWS)
+			Make/FREE/N=(numIndices) out = NaN
+			for(i = 0; i < numIndices; i += 1)
+				WAVE/Z value = GetLastSetting(numericalValues, sweeps[i], str, UNKNOWN_MODE)
+				if(WaveExists(value))
+					out[i] = value[0]
```

Comment:
Nijet. GetLastSetting returns a wave with `LABNOTEBOOK_LAYER_COUNT` rows. The first 8 (aka `NUM_HEADSTAGES`) rows are for the headstage dependent settings, the last one are for the headstage independent settings. So we need to feed the info into the operation which headstages are meant.

I'll discuss with @timjarsky how the interface of the function should look like.

Reply:
@ukos-git @timjarsky A quick discussion lead to the obvious `labnotebook("entry", channels(...), sweeps())`.